alphabase/peptide/precursor.py

```diff
@@ -126,7 +135,16 @@ def update_precursor_mz(
     return precursor_df
```

Comment:
Do you think we should return None to emphasize that this function is inplace? 

I think it's a bit dangerous if you are inplace and still return a dataframe as this pattern is used in pandas for not-inplace operations.

Reply:
Yes. I have been thinking of this for a long time, I think we should learn pytorch that inplace methods are `xxxx_()` and return None.

Reply:
We shall create a ticket about this in alphaX development projects.

Reply:
I think if we agree on the naming and document this in the design philosophy its fine. I like the xxx_() and return None pattern.